src/GafferSceneUIModule/ToolBinding.cpp

```diff
@@ -64,6 +64,16 @@ using namespace GafferSceneUI;
 namespace
 {
 
+Gaffer::Box2fPlugPtr cropWindowToolPlugWrapper( CropWindowTool &tool )
+{
+	return tool.plug();
```

Comment:
We need a ScopedGILRelease here, since `findCropWindowPlug()` launches computes.

Reply:
Added [here](https://github.com/GafferHQ/gaffer/pull/3882/commits/db5594eeaa52688b5b3b1b153684c235c4672e73).